ui/scss/_studio-view.scss

```diff
@@ -481,3 +489,127 @@ table.scroll-body {
 .studio-view .navbar-fixed-top {
   z-index: 100;
 }
+
+#approve-dialog{ 
+  .aprove-subtitle{
+    font-size: 15px;
+  }
+  .action-wrapper{
+    position: relative;
+    top: 39px;
+  }
+}
+
+.submission-comment{
+  height: 111px!important;
+}
+
+.dependencies-display{
+  height: 444px;
+}
+
+//Publish - soft hard dependencies.
```

Comment:
Slightly unorthodox but do check if you can import "../route/to/the/file/on/the/app/directory"

Reply:
Done